### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by fabgrel10**
In this PR:

- [x] Initialize project.
- [x] Add React Redux.
- [x] Structure application files using a "feature folder" approach and use the ducks pattern for Redux files.
- [x] Add React Router and set two <Route>s and <Link>s for the app's navigation.
- [x] Books is the default view and displays the list of books.
- [x] Books have a form for adding a book (no functionality yet).
- [x] Categories displays "Under construction" text only.

### CODE REVIEW

File: src/components/Books/Books.jsx [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
     <div>
-      <h1>Books</h1>
+      <ul>
+        {books.map(book => (
+          <div key={book.id}>
+            <li>
+              <span>{book.title}</span>
+            </li>
+            <button>Remove</button>
+          </div>
```
Comment (TedLivist) [inline src/components/Books/Books.jsx:18]: - [ ] In keeping with the SOLID principle let's refactor this `Books` component. Your **books** should be rendered using a new **`Book`** (or whatever name you wish to call it) component. Kindly extract line 13 - line 18 (as selected) into the new component and pass in this `id`, `title`, and `author` as props instead.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @fabgrel10,

Good job so far!
✔️ Descriptive documentation
✔️ Nicely modularized code.
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books/Books.jsx (hunk 1 above)
Comment (fabgrel10) [reply src/components/Books/Books.jsx:15]: Agreed @TedLivist refactoring and requesting another review.

**Comment by omar-labana** [comment]
Hi @fabgrel10, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**